client/src/components/DataFiles/DataFilesModals/DataFilesModalTables/DataFilesModalListingTable.jsx

```diff
@@ -145,7 +146,18 @@ const DataFilesModalListingTable = ({
   disabled,
 }) => {
   const { loading, error, params, fetchMore } = useFileListing('modal');
-  const isNotRoot = params.path !== '' && params.path !== '/';
+
+  const matchingSystem = useSelector(
+    (state) => state.systems.storage.configuration.filter((s) => !s.hidden),
+    shallowEqual
+  )?.find((s) => s.system === params.system && s.scheme === params.scheme);
+
+  const homeDir = matchingSystem?.homeDir;
```

Comment:
I wonder - is there a simple way to only show the `homeDir` label if that path is an exact match? If you happen to make a listing on the system [at the root level](https://cep.test/workbench/data/tapis/private/cloud.data), it shows "My Data (Corral)" when listing "/"
![Screen Shot 2023-05-08 at 1 26 35 PM](https://user-images.githubusercontent.com/20326896/236903088-194f5497-4d6e-4334-b84c-1fd8852eb087.png)

Reply:
I made this change and changed it so that it shows the base system name (e.g Frontera) instead of the actual system name (i.e My Data (Work)). This was a simpler change to implement whereas changing it to a '/ ' would have required changes in a few other components as well. let me know if that works

Reply:
Works for me 👍